campusmap/src/not-found/NotFoundPage.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { Link } from 'react-router-dom';
+import { Alert } from 'react-bootstrap';
+
+/**
+ * NotFoundPage element that specifies next steps when a page is not found.
+ */
+const NotFoundPage = (): ReactElement => (
+  <Alert>
```

Comment:
Ah okay I see what you mean now about using an `Alert`. For the sake of semantic reasoning, and since the `Alert` component adds some attributes that pertain to an actual alerting element (like a form validation error), it's probably best that we use a different component.
The easiest one would just be a `Container`, which basically just renders an html `div` with some styling to place it centered on the screen with even margins on both sides.

Reply:
You can use the `Container` as a baseline for many different components. There's no coloring, just the spacing and allowing flexbox layouts. You can also add a margin class so you can get some space away from the top of the page, like this: `<Container className="my-4">` where `m` = margin, `y` = vertical, and `4` = 4*the baseline spacing, which I believe is 6 pixels. I can take you through this Tuesday if you'd like.